x-pack/plugins/data_visualizer/public/application/common/components/document_count_content/total_count_header.tsx

```diff
@@ -28,7 +36,15 @@ export const TotalCountHeader = ({ totalCount }: { totalCount: number }) => {
             ),
           }}
         />
-      </EuiText>
+      </EuiText>{' '}
+      <EuiIconTip
+        content={i18n.translate('xpack.dataVisualizer.searchPanel.randomSamplerMessage', {
+          defaultMessage:
+            'Random sampler is being used for the total document count and the chart. Values shown are estimated. Adjust the slider to a higher percentage for better accuracy, or 100% to exact values.',
```

Comment:
I haven't downloaded this PR to test it, but here's a drive-by suggestion:

```suggestion
            'The chart and total document count use random sampler aggregations, which increase speed at the cost of accuracy. Adjust the accuracy with the slider; for exact values, set it to 100%.',
```
P.S. It's not clear to me from the screenshot where the "slider" is. If it's not close to this tooltip we might need to be more detailed about where to find it.